app/docs/dev/policies/meshtrace.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: Mesh Trace (beta)
+title: MeshTrace (beta)
```

Comment:
```suggestion
title: MeshTrace 
beta: true
```
I think now that we have Jekyll, you might be able to use this frontmatter to get a "beta" badge to display on the page: https://docs.konghq.com/contributing/markdown-rules/#markdown-front-matter

Reply:
This won't work unless the template is configured to render a beta badge (which it's not currently)

Reply:
https://github.com/kumahq/kuma-website/issues